_sass/includes/header-navigation.scss

```diff
@@ -1,8 +1,22 @@
+body.guides,
+body.post {
```

Comment:
@FroMage check out http://localhost:4000/guides/hibernate-orm-guide#properties-to-refine-your-hibernate-orm-configuration on this PR
You will see that the config colors clash with the new light background for guides.

Reply:
Also /guides/all-config is not in light background